src/NuGetTrends.Data/NuGetTrendsContext.cs

```diff
@@ -6,11 +6,11 @@ namespace NuGetTrends.Data
 {
     public class PackageDownload
     {
-        public string PackageId { get; set; }
-        public string PackageIdLowered { get; set; }
+        public string? PackageId { get; set; }
```

Comment:
Maybe `= null!` to avoid marking it nullable? 🤔

Reply:
agreed/lazy